### PR TITLE
About coin fee

### PR DESCRIPTION
**Pull request by lochjin**

### CODE REVIEW

File: params/params_testnet.go [hunk 1]
```diff
@@ -126,4 +127,19 @@ var TestNetParams = Params{
 	HDCoinType: 223,
 
 	//OrganizationPkScript:  hexMustDecode("76a914868b9b6bc7e4a9c804ad3d3d7a2a6be27476941e88ac"),
+
+	// Coins config
+	// TODO The future needs to be redefined or deleted
+	CoinsCfg: &types.CoinConfigs{
```
Comment (dindinw) [inline params/params_testnet.go:133]: 1. For the coinid does not set the `CoinsCfg` explicitly such as the MEER and QIT.  better to create new feeType for them , and add restriction that require all coin need to have a `CoinsCfg` explicitly.
2. For the coinid has `CoinsCfg` such as MET and TER, what happen if set value to zero ? (For example,  set `TER` value is 0, and equalfee),

File: params/params_testnet.go [hunk 2]
```diff
@@ -126,4 +127,19 @@ var TestNetParams = Params{
...
+		&types.CoinConfig{
+			Id:    types.METID,
+			Type:  types.FloorFeeType,
+			Value: types.AtomsPerCoin,
+		},
+		&types.CoinConfig{
+			Id:    types.TERID,
+			Type:  types.EqualFreeType,
```
Comment (dindinw) [inline params/params_testnet.go:141]: Typo. Free

File: core/types/coin.go [hunk 3]
```diff
@@ -0,0 +1,54 @@
+package types
+
+import "fmt"
+
+const (
+	// Greater than or equal to
+	FloorFeeType = 0
+
+	// Strict equality
+	EqualFreeType = 1
```
Comment (dindinw) [inline core/types/coin.go:10]: Typo free

**Review by dindinw — CHANGES_REQUESTED** [review]

File: params/params_testnet.go (hunk 2 above)
Comment (lochjin) [reply params/params_testnet.go]: 70d5cd8

File: core/types/coin.go (hunk 3 above)
Comment (lochjin) [reply core/types/coin.go]: 70d5cd8

File: params/params_testnet.go [hunk 4]
```diff
@@ -126,4 +127,19 @@ var TestNetParams = Params{
 	HDCoinType: 223,
 
 	//OrganizationPkScript:  hexMustDecode("76a914868b9b6bc7e4a9c804ad3d3d7a2a6be27476941e88ac"),
+
```
[the remark this answers is not in the document]
Comment (lochjin) [reply params/params_testnet.go]: 5c948fc

File: params/params_testnet.go (hunk 1 above)
Comment (lochjin) [reply params/params_testnet.go]: 4b98218

**Comment by lochjin** [comment]
> 1.) let QIT to be `EqualFeeType` and value 0, so that all test case can be covered.
> =0 (Qit), >0(Meer), >=1 (MET), =1(TER)
> 2.) Is possible let Meer alway >0 ?, the current config is >=0(Meer), can you change to value = dust value ?

5ad332c

**Review by dindinw — APPROVED** [review]

**Pull request merged**